cmd/server-main.go

```diff
@@ -418,7 +418,7 @@ func serverMain(ctx *cli.Context) {
 		}
 	}
 
-	if !globalCLIContext.Quiet {
+	if !globalCLIContext.Quiet && globalInplaceUpdateDisabled {
```

Comment:
@harshavardhana shouldn't this be a negation?  Looks like `MINIO_UPDATE=on` disables the update now, and not `off`.

Reply:
Created PR #10900

Reply:
Correct @rafi